### PR TITLE
chore(ilp): support tags in fuzz test

### PR DESCRIPTION
**Pull request by glasstiger**
This change includes ILP tags in fuzz testing.
Currently the fuzz test sends only columns to QuestDB.
It also adds quotes around string values.
Previously they were treated as symbols because of the missing quotes.

Current message format used by the test:
```
weather0 location=us-midwestC,terület=europeY,temperature=80.0,humidity=25.0,hőmérséklet=13.0,notes=noteU,ветер=63.0 1465839830102302000
```

New format exercising tags and uses proper string fields:
```
weather0,location=us-midwestC,city=LondonU terület="europeY",temperature=80.0,humidity=25.0,hőmérséklet=13.0,notes="noteU",ветер=63.0 1465839830102302000
```

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by bluestreak01** [comment]
It is not clear from PR description nor tests what is the impact of this PR, why should we merge it?

**Comment by glasstiger** [comment]
> It is not clear from PR description nor tests what is the impact of this PR, why should we merge it?

Changed the description

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

File: core/src/main/java/io/questdb/cairo/TableReader.java [hunk 1]
```diff
@@ -472,7 +471,7 @@ public void reshuffleSymbolMapReaders(long pTransitionIndex) {
         // this is a silly exercise in walking the index
         for (int i = 0; i < columnCount; i++) {
 
-            // prevent writing same entry once
+            // prevent writing same entry more times
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/cairo/TableReader.java:474]:
```suggestion
            // prevent writing same entry more than once
```

**Review by bluestreak01 — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 1 / 1 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableReader.java|1|1|100.00%|

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**